app/src/main/java/org/openobservatory/ooniprobe/test/suite/CircumventionSuite.java

```diff
@@ -43,11 +44,8 @@ public static CircumventionSuite initForAutoRun() {
                 list.add(new Psiphon());
             if (pm == null || pm.isTestTor())
                 list.add(new Tor());
-            /* TODO (aanorbel): Riseup VPN Disabled.
-                The riseupvpn experiment has been quite flaky for quite some time.
-                To be enabled only when test is fixed or removed if deemed necessary.
-                if (pm == null || pm.isTestRiseupVPN())
-                list.add(new RiseupVPN());*/
+            if (pm == null || pm.isTestRiseupVPN())
+                list.add(new RiseupVPN());
```

Comment:
We should run RiseupVPN inside the experimental suite. If this change requires too much churn, I would recommend enabling it in the next release, but it must not run as part of circumvention.

Reply:
Started a PR to effect this https://github.com/ooni/probe-android/pull/625 .

We would need to test properly and merge before the next release